docs/src/Upsell.doc.js

```diff
@@ -116,4 +126,53 @@ card(
   />
 );
 
+card(
+  <Example
+    name="Icon"
+    defaultCode={`
+<Upsell
+  title="Give $30, get $60 in ads credit"
+  message="When your friend spends their first $30 on ads, you’ll earn $60 of ads credit, and they’ll get $30 of ads credit, too"
+  primaryLink={{href: "https://pinterest.com", label:"Send invite"}}
+  dismissButton={{
+    accessibilityLabel: 'Dismiss banner',
+    onDismiss: ()=>{},
+  }}
+  image={<Icon icon="pinterest" accessibilityLabel="Pin" color="darkGray" size={32}/>}
+/>
+`}
+  />
+);
+
+card(
+  <Example
+    name="Image"
+    defaultCode={`
+<Upsell
+  title="Stay healthy and safe"
+  message="Please practice social distancing, and check out our resources for adapting to these times."
+  primaryLink={{href: "https://pinterest.com", label:"Visit"}}
+  dismissButton={{
+    accessibilityLabel: 'Dismiss banner',
+    onDismiss: ()=>{},
+  }}
+  image={
+    <Box
```

Comment:
@norrise120 1 way to avoid this wrapping `Box` is to use a column based approach (please confirm whether this works for design):

```jsx
<Box display="flex" direction="row">
  {image && (
    <Box maxWidth={128} column={3}>
      {image}
    </Box>
  )}
  <Box column={image ? 9 : 12}>Content</Box>
</Box>
```